fleet/postgres/init.go

```diff
@@ -82,7 +82,7 @@ func migrateDB(db *sqlx.DB) error {
 					)`,
 					`CREATE INDEX ON agent_groups (mf_owner_id)`,
 					`CREATE INDEX ON agent_groups USING gin (tags)`,
-					`CREATE VIEW agent_group_membership(agent_groups_id, agent_groups_name, agent_mf_thing_id, agent_mf_channel_id, group_mf_channel_id, mf_owner_id, agent_state) as
+					`CREATE or REPLACE VIEW agent_group_membership(agent_groups_id, agent_groups_name, agent_mf_thing_id, agent_mf_channel_id, group_mf_channel_id, mf_owner_id, agent_state) as
```

Comment:
No, as I mentioned in slack, any [DDL](https://www.postgresql.org/docs/14/ddl.html) requires a migration